Agents/OPFAgent/Dockerfile

```diff
@@ -0,0 +1,43 @@
+# First stage: build war file
+#==================================================================================================
+FROM maven:3.6-openjdk-11-slim as builder
+
+# Copy all files into root's home, including the source, pom file, ./m2 directory and credentials
+ADD . /root
+
+# Populate settings templates with credentials, repo name
+WORKDIR /root/.m2
+# (Note that | rather than / is used as the sed delimiter, since encrypted passwords can contain the former, but not the latter
+RUN sed -i "s|MASTER_PASSWORD|$(mvn --encrypt-master-password master_password)|" settings-security.xml
+RUN sed -i "s|REPO_USERNAME|$(cat ../credentials/repo_username.txt)|;s|REPO_PASSWORD|$(cat ../credentials/repo_password.txt|xargs mvn --encrypt-password)|" settings.xml
+
+# Build
+WORKDIR /root/opf-agent
+RUN --mount=type=cache,target=/root/.m2/repository mvn package
```

Comment:
is "--mount=type=cache,target=/root/.m2/repository" to avoid needing to provide credentials?

Reply:
Sorry I didn't commit the credentials folder! This line is from the example Dockerfile in Deploy > examples > java_agent